introduction.tex

```diff
@@ -26,6 +26,8 @@ \section{Terminology}
 A single RISC-V core contains one or more hardware threads, called
 \emph{harts}.
 
+\emph{DXLEN} of a hart is the effective XLEN in M-mode, at reset.
```

Comment:
Is this assuming that XLEN in M-mode at reset is the largest possible XLEN (perhaps MXLEN can be increased or SXLEN can be greater than XLEN in M-mode at reset).

The fact that the priv spec makes no rules about these things is certainly complicating our spec, but we should make our assumptions clear.

Reply:
The privspec says: "The MXL field is always set to the widest supported ISA variant at reset."

But we might as well be explicit here. I'll change that.

Reply:
ah, if it says that, excellent. I think this is good then.